src/core/sys/openbsd/pthread_np.d

```diff
@@ -0,0 +1,23 @@
+/**
+  * D header file for OpenBSD pthread_np.h.
+  *
+  * Copyright: Copyright © 2021, The D Language Foundation
+  * License: <a href="http://www.boost.org/LICENSE_1_0.txt">Boost License 1.0</a>.
+  * Authors: Brian Callahan
+  */
+module core.sys.openbsd.pthread_np;
+
+version (OpenBSD):
+extern (C):
+nothrow:
+@nogc:
+
+public import core.sys.posix.sys.types;
+public import core.sys.posix.signal; // for stack_t
```

Comment:
The other BSD variants don't publicly import this module; I guess non-publicly importing `stack_t` alone would be better wrt. consistent namespace pollution.

Reply:
I do plan to use this module in one place in druntime after this is committed.
I switched `public import core.sys.posix.signal; // for stack_t` to `private import core.sys.posix.signal : stack_t;` in a force push.

Reply:
Thx - `private` is the default and so not needed; there are only half a dozen other occurrences of `private import` in druntime/Phobos, probably ancient.

Reply:
Alright; I'll remove the private keyword. Thanks.

Reply:
(Done)